src/AspNetCoreModuleV2/AspNetCore/src/proxymodule.cxx

```diff
@@ -152,16 +152,6 @@ ASPNET_CORE_PROXY_MODULE::OnExecuteRequestHandler(
 
     m_pApplicationInfo->ExtractApplication(&pApplication);
 
-    // make sure application is in running state
```

Comment:
Why was this removed?

Reply:
We shouldn't be checking the status of the application in the shim. This caused an IIS 500 to be returned instead of ours.

Reply:
Should we bring this back since we return IAPPLCIATION instance now?